src/PowerShellEditorServices/Services/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -99,12 +103,11 @@ public CodeLens[] ProvideCodeLenses(ScriptFile scriptFile)
             {
                 if (symbol is PesterSymbolReference pesterSymbol)
                 {
-                    if (pesterSymbol.Command != PesterCommandType.Describe)
+                    if (_configurationService.CurrentSettings.Pester.Pester5CodeLens 
```

Comment:
Could these conditions be combined into the above, so as to only have 1 `if`?

Reply:
I tried that and it looks terrible. Would it be better to do `if (!) { continue }`? That would reduce levels of nested ifs as well.

Reply:
Ha! That would be my preference too

Reply:
Done.